pkg/virt-controller/services/template_test.go

```diff
@@ -1078,7 +1078,7 @@ var _ = Describe("Template", func() {
 				Expect(pod.Spec.Containers[1].Image).To(Equal("some-image:v1"))
 				Expect(pod.Spec.Containers[1].ImagePullPolicy).To(Equal(k8sv1.PullPolicy("IfNotPresent")))
 				Expect(pod.Spec.Containers[1].VolumeMounts[0].MountPath).To(Equal(hooks.HookSocketsSharedDirectory))
-
+				Expect(pod.Spec.Containers[1].VolumeMounts[0].SubPath).To(Equal("hook-sidecar-0"))
```

Comment:
IIUC, this test tests node selectors.
Is there currently a dedicated test for a VMI with sidecars?

Reply:
you are right, 
yes (unit test), in the same file, will update that one instead (+ will also add the env var UT you asked on other comment) there